main.py

```diff
@@ -0,0 +1,266 @@
+  
+  if userInput in gen1 and userInput != userInput2:
+    print("Elizabeth")
+  
+  if userInput in gen2 and userInput != userInput2:
+    if userInput == marj:
+      print("Marj")
+      
+    if userInput == rosanne:
+      print("Rosanne")
+  
+    if userInput == ruthanne:
+      print("Ruthanne")
+  
+    if userInput == donnie:
+      print("Donnie")
+  
+    if userInput == patricia:
+      print("Patricia")
+  
+  if userInput in gen3 and userInput != userInput2:
+    if userInput == kim:
+      print("Kim")
+  
+    if userInput == monica:
+      print("Monica")
+  
+    if userInput == martin:
+      print("Martin")
+  
+    if userInput == gabe:
+      print("Gabe")
+  
+    if userInput == john:
+      print("John")
+  
+    if userInput == ben:
+      print("Ben")
+  
+    if userInput == jerome:
+      print("Jerome")
+  
+  if userInput in gen4 and userInput != userInput2:
+    if userInput == johnG:
+      print("John G")
+  
+    if userInput == jamie:
+      print("Jamie")
+  
+    if userInput == julia:
+      print("Julia")
+  
+    if userInput == phil:
+      print("Phil")
+  
+    if userInput == alley:
+      print("Alley")
+  
+    if userInput == anna:
+      print("Anna")
+  
+    if userInput == steven:
+      print("Steven")
+  
+    if userInput == ellane:
+      print("Ellane")
+  
+    if userInput == jake:
+      print("Jake")
+  
+    if userInput == veera:
+      print("Veera")
+  
+    if userInput == precila:
+      print("Precila")
+  
+    if userInput == clarence:
+      print("Clarence")
+  
+  if userInput in gen5 and userInput != userInput2:
+    if userInput == lincoln:
+      print("Lincoln")
+  
+    if userInput == mo:
+      print("Mo")
+  
+  if userInput in gen6 and userInput != userInput2:
+    print("Alex")
+  
+  ######################################################
+    
+  # If both members are in generation 2, they are siblings 
+  if userInput in gen2 and userInput2 in gen2 and userInput != userInput2:
+    print("is the sibling of")
+  
+  # If both members are in generation 3, they are siblings 
+  if userInput in gen3 and userInput2 in gen3 and userInput != userInput2:
+    print("is the sibling of")
+  
+  # Checks the relationship using math
+  if -2 < userInput - userInput2 < 0 and userInput and userInput2 not in hasNoChildren: # Gen 1
+    print("is the parent of")
+  if -6 < userInput - userInput2 < -1: # Gen 2
+    print("is the parent of")
+  if -13 < userInput - userInput2 < -5: # Gen 3
+    print("is the parent of")
+  if userInput - userInput2 < -12: # Gen 4
+    print("is the parent of")
+  if userInput - userInput2 < -24: # Gen 5
+    print("is the parent of")
+  if userInput - userInput2 < -26: # Gen 6
+    print("is the parent of")
+    
+  ######################################################
+  
+  if userInput2 in gen1 and userInput != userInput2:
+    print("Elizabeth")
+  
+  if userInput2 in gen2 and userInput != userInput2:
+    if userInput2 == marj:
```

Comment:
If you see yourself repeating code this much, something is wrong.  Good software practices follows the DRY principle.